source/features/quick-mention.tsx

```diff
@@ -33,12 +33,20 @@ function mentionUser({delegateTarget: button}: delegate.Event): void {
 }
 
 function init(): void {
-	// `:first-child` avoids app badges #2630
-	// The hovercard attribute avoids `highest-rated-comment`
-	// Avatars next to review events aren't wrapped in a <div> #4844
-	const avatars = select.all(`:is(div.TimelineItem-avatar > [data-hovercard-type="user"]:first-child, a.TimelineItem-avatar):not([href="/${getUsername()!}"], .rgh-quick-mention)`);
+	const avatars = select.all(`:is(${[
```

Comment:
I'd probably extract this array instead to avoid <code>(`:is(${[</code>

Common `is`-less example: 

```js
const avatarSelector = [

].map(x => x + ':not()');
const avatars = select.all(avatarSelector);
```

Reply:
You could also move this to the lint PR if you prefer

Reply:
Latest commit did not match this, let's merge this PR without it and you can make the suggested change in the lint PR if desired.